src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -796,8 +796,16 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   /**
    * Handles 'Done Editing' click event.
    */
-  doneEditingHandler(): void {
-    this.router.navigateByUrl('/web/instructor/sessions');
+  doneEditingHandler(modal: any): void {
+    if (this.questionEditFormModels.some((questionEditFormModel: QuestionEditFormModel): boolean => {
+      return questionEditFormModel.isEditable;
```

Comment:
If there is just one return statement, you can omit the curly bracket. `.some(q => q.isEditable)`